File: app/src/main/java/ru/visualmath/android/api/model/Module.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Module implements Serializable {

    @SerializedName("_id")
    private String id;

    @SerializedName("name")
    private String name;

    @SerializedName("content")
    private String content;

    @SerializedName("images")
    private List<String> images;

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getContent() {
        return content;
    }

    public List<String> getImages() {
        return images;
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/Module.kt
package ru.visualmath.android.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Module(

        @SerializedName("_id")
        val id: String = "",

        @SerializedName("name")
        val name: String = "",

        @SerializedName("content")
        val content: String = "",

        @SerializedName("images")
        val images: List<String> = emptyList()
) : Parcelable
